packages/devtools_app/lib/src/inspector/inspector_service.dart

```diff
@@ -10,13 +10,19 @@ import 'dart:async';
 import 'dart:convert';
 import 'dart:developer';
 
+import 'package:devtools_app/src/octicons.dart';
+import 'package:flutter/cupertino.dart';
```

Comment:
This class couldn't really operate without package:flutter so might as well take advantage of package:flutter for handy constants. We now use the unittest runner to run this test on the regular flutter bots.